@xen-orchestra/lite/src/components/tasks/TasksTable.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <UiTable class="tasks-table">
+    <template #header>
+      <th>{{ $t("task.name") }}</th>
+      <th>{{ $t("task.object") }}</th>
+      <th>{{ $t("task.progress") }}</th>
+      <th>{{ $t("task.started") }}</th>
+      <th>{{ $t("task.estimated-end") }}</th>
+    </template>
+
+    <PendingTaskRow
+      v-for="task in pendingTasks"
+      :key="task.uuid"
+      :task="task"
+    />
+    <FinishedTaskRow
+      v-for="task in finishedTasks"
+      :key="task.uuid"
+      :task="task"
+    />
```

Comment:
IMO, since these two components are identical, we should mutualize them.
```js
  <TaskRow
      v-for="task in pendingTasks"
      :key="task.uuid"
      :task="task"
    />
  <TaskRow
    class="finished-tasks"
    v-for="task in finishedTasks"
    :key="task.uuid"
    :task="task"
  />
```

```css
.finished-tasks :deep(td) {
  opacity: 0.5;
}
```

Reply:
I mutualized the components